core/templates/components/shared-component.module.ts

```diff
@@ -135,11 +142,34 @@ const toastrConfig = {
     ObjectComponentsModule,
     SharedFormsModule,
     SharedPipesModule,
+    TranslateModule.forRoot({
+      defaultLanguage: 'en',
+      loader: {
+        provide: TranslateLoader,
+        useFactory: (TranslateLoaderFactory.createHttpLoader),
+        deps: [HttpClient],
+      },
+      parser: {
+        provide: TranslateParser,
+        useClass: TranslateCustomParser
+      }
+    }),
+    TranslateCacheModule.forRoot({
+      cacheService: {
+        provide: TranslateCacheService,
+        useFactory: TranslateCacheFactory.createTranslateCacheService,
+        deps: [TranslateService, TranslateCacheSettings]
+      },
+      cacheName: 'NG_TRANSLATE_LANG_KEY',
```

Comment:
same cookie name as angular-translate. So, both angular and angularjs share the same cookie for caching.